test/load/cmd/root_cmd.go

```diff
@@ -0,0 +1,302 @@
+)
+
+var (
+	lock      = sync.RWMutex{}
+	usernames = []string{}
+)
+
+// run creates independent go routines which take care of
+func run(cmd *cobra.Command, args []string) error {
+	counter = startAt
+	client, config, err := newClient()
+	if err != nil {
+		return errors.Wrap(err, "unable to initialize the client")
+	}
+	// confirm connection to cluster defined in config
+	if confirm, err := confirmCluster(config); err != nil {
+		return err
+	} else if !confirm {
+		return nil
+	}
+
+	// user signup routine
+	signupChan := make(chan int)
+	errorChan := make(chan error)
+	go enrollUsers(cmd, client, signupChan, errorChan)
+	// user deactivation routine
+	deactivationChan := make(chan int)
+	go deactivateUsers(cmd, client, deactivationChan, errorChan)
+	// user banning routine
+	banChan := make(chan int)
+	go banUsers(cmd, client, banChan, errorChan)
+
+	go displayProgress(cmd, signupChan, deactivationChan, banChan, errorChan)
+
+	// wait until user kills with `<Ctrl+C>` this program
+	// See https://gobyexample.com/signals
+	fmt.Fprintf(cmd.OutOrStdout(), "🚛 load started. Press <ctrl>+C to stop...\n")
+	sigs := make(chan os.Signal, 1)
+	done := make(chan bool, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+	go func() {
+		<-sigs
+		done <- true
+	}()
+	<-done
+	return nil
+}
+
+func confirmCluster(config *restclient.Config) (bool, error) {
+	prompt := promptui.Select{
+		Label: "☁️ connecting to https://" + config.Host,
+		Items: []string{
+			"No, not this one!",
+			"Yes, use https://" + config.Host,
+		},
+	}
+	i, _, err := prompt.Run()
+	if err != nil {
+		return false, err
+	}
+	return !(i == 0), nil
+}
+
+var counter = 0
+
+func enrollUsers(cmd *cobra.Command, client *rest.RESTClient, signupChan chan int, errorChan chan error) {
+	time.Sleep(signupUserAfter)
+	for {
+		if username, err := enrollUser(client); err != nil {
+			errorChan <- errors.Wrapf(err, "failed to create user '%s'", username)
+		}
+		signupChan <- 1
+		counter++
+		time.Sleep(signupUserInterval)
+	}
+}
+
+func enrollUser(client *restclient.RESTClient) (string, error) {
+	username := fmt.Sprintf("test-user-%06d", counter)
+	userEmail := username + "@example.com"
+	md5hash := md5.New()
+	// Ignore the error, as this implementation cannot return one
+	_, _ = md5hash.Write([]byte(userEmail))
+	emailHash := hex.EncodeToString(md5hash.Sum(nil))
+	usersignup := toolchainv1alpha1.UserSignup{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: hostOperatorNS,
+			Name:      username,
+			Annotations: map[string]string{
+				toolchainv1alpha1.UserSignupUserEmailAnnotationKey:           userEmail,
+				toolchainv1alpha1.UserSignupVerificationCounterAnnotationKey: "0",
+			},
+			Labels: map[string]string{
+				toolchainv1alpha1.UserSignupUserEmailHashLabelKey: emailHash,
+			},
+		},
+		Spec: toolchainv1alpha1.UserSignupSpec{
+			Username:             username,
+			Approved:             true,
+			VerificationRequired: false,
+		},
+	}
```

Comment:
Looking at this I think that we should start using some unified function for creating the `UserSignups` - it's yet another place where we create the `UserSignup` with all annotations & labels and it's pretty hard to keep it consistent. 
We could maybe place it in api repo, WDYT?
Can be done as separate PR(effort) of course :wink: